src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson.txt

```diff
@@ -157,7 +157,7 @@ Table({Value:Blank()},{Value:Blank()},{Value:Blank()})
 Table({Value:5},{Value:Blank()},{Value:7})
 
 >> ForAll(ParseJSON("[""hi"", null, ""hello""]"), Text(ThisRecord))
-Table({Value:"hi"},{Value:Blank()},{Value:"hello"})
+Table({Value:"hi"},{Value:""},{Value:"hello"})
```

Comment:
Conversion functions (such as Text) should preserve blank/null values

Reply:
@gregli-msft is this true? In our chat, I remember that we talked about making this more consistent with the existing Text function which maps Blank() to ""